mkdocs.yml

```diff
@@ -1,3 +1,4 @@
 site_name: djinni-support-lib
 nav:
-  - "install.md"
\ No newline at end of file
+  - install.md
+  - developer-guide.md
```

Comment:
additional documentation pages like `python-support.md` need to be added in `mkdocs.yml` to be included in the final documentation.

Reply:
You can render a preview of the documentation by running `mkdocs serve` ([more information](https://www.mkdocs.org/#getting-started)).